patches/tModLoader/Terraria/ModLoader/UI/UIModInfo.cs

```diff
@@ -183,6 +214,15 @@ internal class UIModInfo : UIState
 			Utils.OpenToURL(_url);
 		}
 
+		private void VisitModSteamPage(UIMouseEvent evt, UIElement listeningElement) {
+			SoundEngine.PlaySound(10);
+			if (Social.Steam.WorkshopHelper.ModManager.SteamUser && Steamworks.SteamUtils.IsOverlayEnabled() && (Main.graphics.IsFullScreen || Main.graphics.PreferredBackBufferWidth >= 1920 || Main.graphics.PreferredBackBufferHeight >= 1080))
+				Steamworks.SteamFriends.ActivateGameOverlayToWebPage("http://steamcommunity.com/sharedfiles/filedetails/?id=" + _publishedFileId, Steamworks.EActivateGameOverlayToWebPageMode.k_EActivateGameOverlayToWebPageMode_Modal);
+			else
+				Utils.OpenToURL("http://steamcommunity.com/sharedfiles/filedetails/?id=" + _publishedFileId);
```

Comment:
Hmm, debate this one. Don't follow the logic as the overlay isn't always preferable; at low resolutions, the web browser built into Steam Overlay is pretty pathetic. Doesn't scale well since the overlay browser lacks a zoom feature, as an example:
![image](https://user-images.githubusercontent.com/59670736/130717486-54f1dba6-90cf-4a0f-b3c1-ee207d3a6bea.png)

I should have included a comment explaining that in the code.

And all the checks for verifying the user is logged in occurs elsewhere; no reason why viewing info about a mod should require yet another check, so not following the "require logged in to Steam logic", SteamAPI.Init() is responsible for that in the first place.

Reply:
The lowest resolution we should care about is 1280x720. If it looks at least barely usable in that - overlay should be preferred.

Reply:
More than usable:
![image](https://user-images.githubusercontent.com/6795251/131268000-eed1a9e8-1e03-4ed4-84e6-2f035b40dbe8.png)